src/node-lib/curriculum-api-2023/queries/pupilLessonOverview/pupilLessonOverview.gql

```diff
@@ -3,7 +3,7 @@ query pupilLessonOverview(
   $lessonSlug: String!
   $unitSlug: String!
 ) {
-  lesson: published_mv_lesson_overview_3_0_0(
+  lesson: published_mv_lesson_overview_3_0_1(
```

Comment:
I think we can safely move to this query?

Reply:
yes this allows us to distinguish legacy from non-legacy lessons